src/Dfc.CourseDirectory.Web/Startup.cs

```diff
@@ -190,12 +188,6 @@ public void ConfigureServices(IServiceCollection services)
             //TODO
             //services.Configure<GoogleAnalyticsOptions>(options => Configuration.GetSection("GoogleAnalytics").Bind(options));
 
-
-            // Register the background worker helper
-            services.AddHostedService<QueuedHostedService>();
-            services.AddSingleton<IBackgroundTaskQueue, BackgroundTaskQueue>();
-            services.Configure<FormOptions>(x => x.ValueCountLimit = 10000);
```

Comment:
I don't think this will ever be resolved as `FormOptions` has already been registered.

Reply:
You can have multiple `.Configure()` calls and each of their `configureOptions` will be called in order.